### PR TITLE
Add task in space room with no assignee

### PR DESCRIPTION
**Pull request by esmaoui**

### CODE REVIEW

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/task/TaskChatInSpaceRoomTestIT.java [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+        homePagePlatform.goToMySpaces();
+        spaceManagement.addNewSpaceSimple(space, space, 6000);
+        homePagePlatform.goToChat();
+        switchTo().window(1);
+        ELEMENT_CONTACT_LIST.find(byText(space)).click();
+        ELEMENT_COLLABORATION_ACTIONS.click();
+        ELEMENT_CHAT_CREATE_TASK.click();
+        ELEMENT_CHAT_TASK_NAME.setValue(task);
+        ELEMENT_CHAT_POST_TASK_BUTTON.click();
+        ELEMENT_CONTAINER_LIST_MESSAGES.find(byLinkText(task)).shouldBe(Condition.visible);
```
Comment (ilyeskallala) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/task/TaskChatInSpaceRoomTestIT.java]: you can create a function createTaskInChat() to reuse it. (same for other tests)

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/task/TaskChatInSpaceRoomTestIT.java [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+        switchToParentWindow();
+        homePagePlatform.goToTaskPage();
+        ELEMENT_LIST_PROJECT.find(byText(space)).click();
+        ELEMENT_TABLE_PROJECT.parent().parent().parent().find(byText(task)).should(Condition.exist).click();
+        ELEMENT_ASSIGNEE_TASK.shouldHave(Condition.text(DATA_NAME_USER1));
```
Comment (ilyeskallala) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/task/TaskChatInSpaceRoomTestIT.java]: you can create a function checkTask() .

**Review by ilyeskallala — CHANGES_REQUESTED** [review]

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/task/TaskChatInSpaceRoomTestIT.java (hunk 2 above)
Comment (esmaoui) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/task/TaskChatInSpaceRoomTestIT.java]: done

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/task/TaskChatInSpaceRoomTestIT.java (hunk 1 above)
Comment (esmaoui) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/task/TaskChatInSpaceRoomTestIT.java]: done

**Pull request closed without merging**